### DISCUSSION TITLE
What is the recommended tool for pressure testing moosefs？

### DISCUSSION THREAD

**Discussion by cuibaiyi** [moosefs/moosefs#396] [Q&A]
#### What is the recommended tool to test the throughput of moosefs? What configuration can be used to optimize or achieve the maximum TPS of moosefs.

**Comment by jSML4ThWwBID69YC** [comment]
Try using FIO for testing. See https://fio.readthedocs.io/en/latest/fio_doc.html

**Comment by asomers** [comment]
Has anybody ever considered writing a fio plugin for moosefs, to bypass the FUSE layer?

**Comment by cuibaiyi** [comment]
谢谢啦
